### ISSUE TITLE
Making the outline command view state dependent is confusing

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode#133953]
Testing #133901

Hiding the command is confusing since it force me to think about whether the outline view is visible before I open the command palette. Since we are not saving anything I would always include the command.

**Comment by chrisdias** [comment]
+1 on being confusing. In fact, I can't figure out how to make the command available in the Command Palette :)

**Comment by devinvalenciano** [comment]
I agree - I feel that clicking Outline when the Outline view is showing should toggle the view

**Comment by IanMatthewHuff** [comment]
Three votes is plenty for me :). I'll make a change to make sure the command palette command is always available when in the context of a Jupyter notebook and not regard the visibility state of the outline view.

@devinvalenciano It's interesting, I'm not as familiar with all the core vscode commands, but I'm not currently seeing a way to turn the button into a toggle. For the view I can focus it, toggle its overall visibility (which totally hides the control), collapse items within it, but not a way to collapse the control down. I'll follow up to see if I'm missing a way to do this, but at least for this release the button might just have to stay as it is for now (enabled / disabled).

**Comment by IanMatthewHuff** [comment]
Actually after playing with it a bit I'm going to make both the command palette command and the outline button available at all times that a notebook is active, never disabled. Even if the outline view is open it's helpful that the button brings keyboard focus to that area. There are other ways around that with keyboard navigation, but it seems wrong if that is part of a user's workflow to disable the button just because the view is already visible.
